runtimevar/etcdvar/etcdvar.go

```diff
@@ -58,7 +58,10 @@ func (w *watcher) WatchVariable(ctx context.Context, prevVersion interface{}, pr
 
 	// Create a watching channel in case the variable hasn't changed.
 	// We must create it now before the Get to avoid race conditions.
-	ch := w.client.Watch(ctx, w.name)
+	var ch clientv3.WatchChan
```

Comment:
Shouldn't there be an additional conformance test for the changes in this pull request?

Reply:
No, there's no change in behavior, we're just avoiding some unnecessary work.